### ISSUE TITLE
sqlsrv driver not available with php 7.1

### ISSUE THREAD

**Issue by abhimanyu003** [laravel/homestead#715]
## Versions
  - Vagrant: 2.0.0.
  - Provider: Virtualbox
  - Homestead: 6.5.0

There is no sqlsrv driver available with php7.1 **but it if we use php7.2 it is working fine.**

## Host operating system
4.11.12-1-rt16-MANJARO

## Homestead.yaml

**No sqlsrv driver**
```yaml
sites:
    - map: homestead.test
      to: /home/vagrant/code/homestad/public
```

**sqlsrv driver available**

```yaml
sites:
    - map: homestead.test
      to: /home/vagrant/code/homestad/public
      php: "7.2"
```

**Comment by svpernova09** [comment]
There is some underlying issues around `sqlsrv` driver. According to https://github.com/Microsoft/msphpsql they're supporting 7.0, 7.1. I threw support in via settler for 7.2 just to see if it'd even work. Amusing that it does, but I'm unsure of why it's breaking with 7.1. I've been building test boxes of settler lately and this driver is being a huge pain. 

Show a full output of `vagrant destroy && vagrant up` and maybe we'll see an indicator there. at the moment I don't have a solution for this as I'm still debugging it in the upstream Settler box.

**Comment by abhimanyu003** [comment]
Yes the `sqlsrv` driver is working 100% fine with PHP 7.2.
There is nothing suspicious with `vagrant destory && vagrant up` to share.

Somehow `sqlsrv` driver is available with PHP 7.1 or PHP 7.0 before `4.0` updates.

**Comment by svpernova09** [comment]
Closing as related to #719 

```
I've spent all day trying to get a solution for this an am currently unable to get a working resolution. I'm declaring MSSQL drivers unsupported until the documentation is updated to provide a working solution or until someone is able to PR a working solution. Sorry for the issues.

https://github.com/laravel/settler/pull/125
```
